documentation/bin/createDocu.groovy

```diff
@@ -22,9 +23,8 @@ class TemplateHelper {
         parameters.keySet().toSorted().each {
 
             def props = parameters.get(it)
-
-            def defaultValue = isComplexDefault(props.defaultValue) ? renderComplexDefaultValue(props.defaultValue) :
-                                props.defaultValue != null ? "`${props.defaultValue}`" : ''
+  
+            def defaultValue = isComplexDefault(props.defaultValue) ? renderComplexDefaultValue(props.defaultValue) : "${props.defaultValue ? ('`' + props.defaultValue + '`') : ''}"
```

Comment:
check for `null` values only

Reply:
@marcusholl: If I remember this correct, we fixed the same issue 2 days ago. If `props.defaultValue` has the value `false` it is not printed.

Reply:
@CCFenner ... got it.